packages/addon-dev/src/rollup-app-reexports.ts

```diff
@@ -46,7 +46,7 @@ export default function appReexports(opts: {
           });
         }
       }
-      let originalAppJS = pkg['ember-addon']?.['app-js'];
+      let originalAppJS = pkg['ember-addon']?.['app-js'] ?? {};
```

Comment:
?? {} is needed here because 2 lines after this we call Object.keys... and we can't call Object.keys on falsey values